### ISSUE TITLE
Allow hiding for screen readers

### ISSUE THREAD

**Issue by MatzeKitt** [epiphyt/block-control#24]
### What feature are you requesting?

Add option to hide content from screen readers only via `aria-hidden` attribute.

### Why is this feature necessary?

Some features are not accessible and thus can be disabled for screen readers.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
